app/models/models.scala

```diff
@@ -130,3 +132,18 @@ object LaunchConfiguration {
   import utils.DateUtils._
   implicit val jsonFormat = Json.format[LaunchConfiguration]
 }
+
+case class Email(address: String, subject: String, message: String)
```

Comment:
yes there is `SendEmailRequest`
It has the following ctor:
```
public SendEmailRequest(String source, Destination destination, Message message)
```
We could move the stuff that are being done in the `AWSMailClient.send` function and actually move around a `SendEmailRequest class. The thought behind creating the `Email` class was that we might change (?) mail client library in the future so it would be better to have generic (and custom made) classes being moved around...but maybe this is not very probable!